packages/obconf.rb

```diff
@@ -2,17 +2,19 @@
 
 class Obconf < Package
   description 'ObConf - Openbox configuration tool'
-  homepage 'https://openbox.org/wiki/ObConf:About'
+  homepage 'http://openbox.org/wiki/ObConf:About'
   version '2.0.4'
   compatibility 'all'
-  source_url 'https://openbox.org/dist/obconf/obconf-2.0.4.tar.gz'
+  source_url 'http://openbox.org/dist/obconf/obconf-2.0.4.tar.gz'
   source_sha256 '71a3e5f4ee246a27421ba85044f09d449f8de22680944ece9c471cd46a9356b9' 
 
   depends_on 'gtk2'
+  depends_on 'libglade'
   depends_on 'openbox'
 
   def self.build
-    system "./configure"
+    system "./configure",
+           "--mandir=/usr/local/share/man"
```

Comment:
We need `#{CREW_OPTIONS}` to abstract the directories in case someone wants to install in another location besides `/usr/local`.

Reply:
I will also add the `--prefix=#{CREW_PREFIX}` option to support non-standard crew prefix